File: CRUD/Java usando NetBeans/Aplicacion Web usando MVC Spring V2/SistemaWebSpring/src/main/java/sistema/functions/Conexion.java
// Written by Ismael Heredia in the year 2017

package sistema.functions;

import java.sql.Connection;
import java.sql.DriverManager;

public class Conexion {

    public Connection con;

    public void abrir() {
        try {
            Class.forName("com.mysql.jdbc.Driver");
            con = DriverManager.getConnection("jdbc:mysql://localhost/sistemav2", "root", "");
        } catch (Exception e) {
            System.out.println(e);
        }
    }

    public void cerrar() {
        if (con != null) {
            try {
                con.close();
            } catch (Exception e) {
                //
            }
        }
    }

    public Connection retornar() {
        return con;
    }

}

File: CRUD/C#/Aplicacion con SQL Server/aplicacion/Conexion.cs
// Written By Ismael Heredia in the year 2016

using System;
using System.Collections.Generic;
using System.Text;
using System.Data;
using System.Data.SqlClient;

namespace aplicacion
{
    class Conexion
    {
        public SqlConnection conexion;
        public SqlCommand comando;
        public SqlDataReader dr;
        public DataTable dt;

        public Conexion()
        {
            conexion = new SqlConnection();
            comando = new SqlCommand();
            dt = new DataTable();
            dr = null;
        }

        public void abrir()
        {
            conexion.ConnectionString = "Data Source=localhost\\SQLEXPRESS;Initial Catalog=sistema;User ID=admin;Password=123456";
            conexion.Open();
            comando.Connection = conexion;
            comando.CommandType = CommandType.Text;
        }
        public void cerrar()
        {
            conexion.Close();
            conexion.Dispose();
        }
    }
}
